### PR TITLE
Feature: Ability to adjust opacity of players near shared & guessed egg waypoints

### PR DESCRIPTION
**Pull request by RobotHanzo**
## What
Feature: Ability to adjust opacity of players near shared & guessed egg waypoints (range: 4 blocks)

<details>
<summary>Images</summary>
Default opacity: 100

image

</details>

## Changelog New Features
+ Ability to adjust the opacity of players near shared and guessed egg waypoints. - RobotHanzo

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/config/features/event/HoppityEggsConfig.java [hunk 1]
```diff
@@ -33,6 +34,11 @@ public class HoppityEggsConfig {
     @FeatureToggle
     public boolean sharedWaypoints = true;
 
+    @Expose
+    @ConfigOption(name = "Adjust player opacity", desc = "Adjust the opacity of players near shared & guessed egg waypoints.")
+    @ConfigEditorSlider(minValue = 0, maxValue = 255, minStep = 1)
+    public int playerOpacity = 100;
```
Comment (martimavocado) [inline src/main/java/at/hannibal2/skyhanni/config/features/event/HoppityEggsConfig.java]: might be more user friendly for this to be from 1-100, and convert that later on into 1-255 so people can look at it more as a percentage

**Review by hannibal002 — CHANGES_REQUESTED** [review]
The code looks fine, also works in game!
Please make `sharedEggLocation` and `possibleEggLocations` accessible outside the Egg Locator class, and move the new functions into a new class for maintainability.

File: src/main/java/at/hannibal2/skyhanni/config/features/event/HoppityEggsConfig.java (hunk 1 above)
Comment (RobotHanzo) [reply src/main/java/at/hannibal2/skyhanni/config/features/event/HoppityEggsConfig.java]: That is true, done!

**Comment by RobotHanzo** [comment]
> The code looks fine, also works in game! Please make `sharedEggLocation` and `possibleEggLocations` accessible outside the Egg Locator class, and move the new functions into a new class for maintainability.

Done

**Comment by hannibal002** [comment]
next beta, sry

**Comment by hannibal002** [comment]
Well, too late for this feature now.
will test in next spring

**Review by hannibal002 — APPROVED** [review]

**Review by hannibal002 — COMMENTED** [review]
miss click :/

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**